Pod/Classes/SEGAdobeIntegration.h

```diff
@@ -9,14 +9,27 @@
 #import <Foundation/Foundation.h>
 #import <Analytics/SEGIntegration.h>
 #import <AdobeMobileSDK/ADBMobile.h>
+#import "ADBMediaHeartbeat.h"
+#import "ADBMediaHeartbeatConfig.h"
 
+@protocol SEGADBMediaHeartbeatFactory <NSObject>
```

Comment:
This makes it public. I don't think we want to expose this to customers.

Reply:
But this is ok for now, doesn't need to block this.

Reply:
Don't i need to define this here though to define the property `@property (nonatomic, strong, nullable) id<SEGADBMediaHeartbeatFactory> ADBMediaHeartbeatFactory;`